docs/datadog_ndk_module.md

```diff
@@ -0,0 +1,31 @@
+# Datadog NDK extension package
+
+Provides useful features for applications which include C/C++ code in their code base and not only (e.g. the NDKCrashReporterPlugin could be used to reports native level crashes for any application).
```

Comment:
and not only ?

Reply:
Yeah...it is too familiar I will rephrase it.